Makefile

```diff
@@ -1,6 +1,23 @@
 UNIT_TAGS :=  "$(subst :, or ,$(shell awk '{print $2}' tests/unit.tags | paste -s -d: -))"
 INTEGRATION_TAGS := "$(subst :, or ,$(shell awk '{print $2}' tests/integration.tags | paste -s -d: -))"
 
+generate-init:
+	python -m scripts.generate_init
+
+check-generate-init:
+	python -m scripts.generate_init --check
+
+lint: check-generate-init
+	black --check .
+	mypy algosdk
+
+check-setup:
```

Comment:
nit: Just to be consistent with PyTeal, we could rename this `sdist-check` and include this step under `lint`

https://github.com/algorand/pyteal/blob/master/Makefile#L50